Tools/gdb/libpython.py

```diff
@@ -1097,8 +1097,8 @@ def proxyval(self, visited):
             return ProxyAlreadyVisited('(...)')
         visited.add(self.as_address())
 
-        result = tuple(PyObjectPtr.from_pyobject_ptr(self[i]).proxyval(visited)
-                       for i in safe_range(int_from_int(self.field('ob_size'))))
+        result = tuple([PyObjectPtr.from_pyobject_ptr(self[i]).proxyval(visited)
+                        for i in safe_range(int_from_int(self.field('ob_size')))])
```

Comment:
The list comprehension is not necessary and doesn't improve the code. This should stay as-is.